demos/gallery/samples/tree/Dendrogram.js

```diff
@@ -0,0 +1,41 @@
+import { Dendrogram } from '@hpcc-js/tree';
+
+var data = {label:"Food",children:[]};
+data.children.push({label:"Fruit",children:[]});
+data.children[0].children.push({label:"Apple",children:[]});
+data.children[0].children[0].children.push({label:"Granny Smith"});
+data.children[0].children[0].children.push({label:"Red Delicious"});
+data.children[0].children[0].children.push({label:"Braeburn"});
+data.children[0].children.push({label:"Orange",children:[]});
+data.children[0].children[1].children.push({label:"Mandarin"});
+data.children[0].children[1].children.push({label:"Tangerine"});
+data.children[0].children[1].children.push({label:"Grapefruit"});
+data.children.push({label:"Vegetable",children:[]});
+data.children[1].children.push({label:"Carrot",children:[]});
+data.children[1].children[0].children.push({label:"Napoli"});
+data.children[1].children[0].children.push({label:"Hercules"});
+data.children[1].children[0].children.push({label:"Bolero"});
+data.children[1].children.push({label:"Lettuce",children:[]});
+data.children[1].children[1].children.push({label:"Romaine"});
+data.children[1].children[1].children.push({label:"Arugula"});
+data.children[1].children[1].children.push({label:"Iceberg"});
+data.children[1].children[1].children.push({label:"Kale"});
+data.children[1].children[1].children.push({label:"Green Leaf"});
+data.children.push({label:"Meat",children:[]});
+data.children[2].children.push({label:"Beef",children:[]});
+data.children[2].children[0].children.push({label:"Chuck"});
+data.children[2].children[0].children.push({label:"Ground"});
+data.children[2].children[0].children.push({label:"Rib"});
+data.children[2].children.push({label:"Duck",children:[]});
+data.children[2].children[1].children.push({label:"Leg"});
+data.children[2].children[1].children.push({label:"Breast"});
+data.children[2].children[1].children.push({label:"Thigh"});
+data.children[2].children.push({label:"Chicken",children:[]});
+data.children[2].children[2].children.push({label:"Leg"});
+data.children[2].children[2].children.push({label:"Breast"});
+data.children[2].children[2].children.push({label:"Thigh"});
```

Comment:
If you export it from the above example you can import it here?

Reply:
How would this be done? Like this?
CircleDendrogram.js
```javascript
const treeData = {label:"root",children:[]};
export treeData;
```
Dendrogram.js
```javascript
import treeData from "./CircleDendrogram.js"
```

Reply:
Or just 
```
export const treeData = ...
```
Having said that, I am not 100% sure its good to have the sample data in a different file from the users point of view?